torch_xla/experimental/pallas_kernels/ragged_paged_attention_v2.py

```diff
@@ -1502,6 +1503,9 @@ def compute_actual_vmem_bytes(num_kv_pages_per_blk):
   elif device_name in TUNED_BLOCK_SIZES:
     if key in TUNED_BLOCK_SIZES[device_name]:
       bkv, bq = TUNED_BLOCK_SIZES[device_name][key]
+    else:
+      logging.warning(
+          f"key({key}) is not in ragged attention kernel's tuning table!")
```

Comment:
nit: when debugging the key not found issue, do you need more info such as the function's args?